src/orion/core/worker/storage_backup.py

```diff
@@ -0,0 +1,473 @@
+    version:
+        (optional) version of experiment to dump
+    """
+    dump_host = os.path.abspath(dump_host)
+
+    # For pickled databases, make sure src is not dst
+    if hasattr(storage, "_db"):
+        orig_db = storage._db
+        if isinstance(orig_db, PickledDB) and dump_host == os.path.abspath(
+            orig_db.host
+        ):
+            raise DatabaseError("Cannot dump pickleddb to itself.")
+
+    dst_storage = setup_storage({"database": {"host": dump_host, "type": "pickleddb"}})
+    logger.info(f"Dump to {dump_host}")
+    _dump(storage, dst_storage, name, version)
+
+
+def load_database(
+    storage, load_host, resolve, name=None, version=None, progress_callback=None
+):
+    """Import data into a database
+
+    Parameters
+    ----------
+    storage: BaseStorageProtocol
+        storage of destination database to load into
+    load_host:
+        file path containing data to import
+        (should be a pickled file representing a PickledDB)
+    resolve:
+        policy to resolve import conflict. Either 'ignore', 'overwrite' or 'bump'.
+        - 'ignore' will ignore imported data on conflict
+        - 'overwrite' will overwrite old data in destination database on conflict
+        - 'bump' will bump imported data version before adding it,
+          if data with same ID is found in destination
+    name:
+        (optional) name of experiment to import (by default, whole file is imported)
+    version:
+        (optional) version of experiment to import
+    progress_callback:
+        Optional callback to report progression. Receives 2 parameters:
+        - step description (string)
+        - overall progress (0 <= floating value <= 1)
+    """
+    load_host = os.path.abspath(load_host)
+
+    # For pickled databases, make sure src is not dst
+    if hasattr(storage, "_db"):
+        dst_db = storage._db
+        if isinstance(dst_db, PickledDB) and load_host == os.path.abspath(dst_db.host):
+            raise DatabaseError("Cannot load pickleddb to itself.")
+
+    src_storage: BaseStorageProtocol = setup_storage(
+        {"database": {"host": load_host, "type": "pickleddb"}}
+    )
+    logger.info(f"Loaded src {load_host}")
+
+    import_benchmarks = False
+    _describe_import_progress(STEP_COLLECT_EXPERIMENTS, 0, 1, progress_callback)
+    if name is None:
+        import_benchmarks = True
+        # Retrieve all src experiments for export
+        experiments = src_storage.fetch_experiments({})
+    else:
+        # Find experiments based on given name and version
+        query = {"name": name}
+        if version is not None:
+            query["version"] = version
+        experiments = src_storage.fetch_experiments(query)
+        if not experiments:
+            raise DatabaseError(
+                f"No experiment found with query {query}. Nothing to import."
+            )
+        if len(experiments) > 1:
+            experiments = sorted(experiments, key=lambda d: d["version"])[:1]
```

Comment:
```suggestion
            experiments = sorted(experiments, key=lambda d: d["version"])[-2:]
```
Should use latest version by default.

Reply:
Ok!